h/schemas/admin_group.py

```diff
@@ -61,6 +82,15 @@ class CreateAdminGroupSchema(CSRFSchema):
         title=_('Description'),
         description=_('Optional group description'),
         validator=colander.Length(max=GROUP_DESCRIPTION_MAX_LENGTH),
-        widget=TextInputWidget(rows=3),
+        widget=TextAreaWidget(rows=3),
+        missing=None
+    )
+
+    origins = colander.SchemaNode(
+        colander.String(),
+        title=_('Scope Origins'),
+        widget=TextAreaWidget(rows=5),
+        hint=_('Enter scope origins (e.g. "http://www.foo.com"), one per line'),
```

Comment:
Using the hint is great. If we could add a placeholder value to the field with an example input that would be even better.